src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -186,15 +186,28 @@ public ObservableCollection<NodeCategoryViewModel> BrowserRootCategories
             get { return LibraryRootCategories; }
         }
 
-        public NodeSearchModel Model { get; private set; }
+        private NodeSearchModel model;
+        public NodeSearchModel Model
+        {
+            get
+            {
+                if (dynamoViewModel != null)
+                    return dynamoViewModel.Model.SearchModel;
+
+                // dynamoViewModel can't be null, this case is just for tests.
+                if (model == null)
+                    model = new NodeSearchModel();
+                return model;
+            }
+        }
+
```

Comment:
Ouch, we shouldn't do this (i.e. shouldn't create `NodeSearchModel` internally which is disconnected from the rest of the system just for testing). Sorry I overlooked the part where unit-tests are still using the second constructor argument, in which case I am proposing the following:

```csharp
internal SearchViewModel(DynamoViewModel dynamoViewModel); // This is for main use case
internal SearchViewModel(NodeSearchModel model); // This is for test
```

Sorry about that!